chsdi/models/bod.py

```diff
@@ -86,6 +86,7 @@ class LayersConfig(Base):
     geojsonUrlit = Column('geojson_url_it', Text)
     geojsonUrlrm = Column('geojson_url_rm', Text)
     geojsonUrlen = Column('geojson_url_en', Text)
+    shop = Column('shop_option_arr', postgresql.ARRAY(Text))
```

Comment:
Okay thanks. Per default we have null vales or empty lists? Best would be `null`.